extensions/aws/processors/PutS3Object.h

```diff
@@ -81,7 +96,7 @@ class PutS3Object : public S3Processor {
   EXTENSIONAPI static constexpr bool SupportsDynamicProperties = true;
   EXTENSIONAPI static constexpr bool SupportsDynamicRelationships = false;
   EXTENSIONAPI static constexpr core::annotation::Input InputRequirement = core::annotation::Input::INPUT_REQUIRED;
-  EXTENSIONAPI static constexpr bool IsSingleThreaded = false;
+  EXTENSIONAPI static constexpr bool IsSingleThreaded = true;
```

Comment:
Can we keep multi-threaded operation?

Reply:
Good catch, I forgot to revert this after discussing it, updated in a63cb4a856a9ed576f994dc849c57a29a8044ae0